### ISSUE TITLE
`available` wraps around wrong

### ISSUE THREAD

**Issue by zvozin** [jmfayard/buildSrcVersions#36]
When the current version is long, newly available version ends up on the next line sans comment marks.

```
    const val services_utils: String = "1.0.1270-kubernetes-SNAPSHOT-1547233874" // available:
            "1.0.1270"
```

**Comment by jmfayard** [comment]
Thanks for the report.

**Comment by jmfayard** [comment]
@zvozin Can you give me the complete dependency string so that I can reproduce and fix the problem?

**Comment by jmfayard** [comment]
should be fixed in version 0.4.0 (un-released yet)
